samples/oauth_rails_vetverification/README.md

```diff
@@ -0,0 +1,24 @@
+# README
+
+This is a sample app that will connect with the oauth provider to access Veteran Verification APIs.
+
+## Running locally
+
+The sample app uses the default SQLite database setup.  The app expects the file `config/application.yml` to contain a key for `va_developer_client_id` and `va_developer_client_secret` using your credentials set up at the [Developer Portal](https://developer.va.gov/apply).  The callback for this app to work needs to be `http://localhost:3000/callback`.
+
+## Using the app
+
+The home page will show an "Oauth Login" button.  That will take you to the VA sign-in page.  You can use a [test login](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/master/test_accounts.md).  Once logged in, you will see the /verify page, which will show the confirmed Veteran Status, Service History, and Disability Ratings for the user.  Trying to access /verify directly (without logging in) will have the user redirected to /login because they don't have a session available in the app.
```

Comment:
I really like that you have links to test users here.. could potentially even add a link in the app. Was it confusing the first time through that the multi-factor auth step is mocked out for you from id.me? That might be worth explaining to a new user.

Reply:
It was, but I was thinking that information would be with the list of test users.  I just realized this is a the same repo, is it okay to update that test user page to mention that?  Or should that be a separate PR?

Reply:
Added a link to test users from the app

Reply:
I added some text to the the `test_accounts.md` file to explain the login process